src/routers/routerV2.ts

```diff
@@ -26,6 +37,13 @@ routerV2.get(
 
 // Speech-to-Text
 routerV2.post('/speech-to-text', analytics, validateApiKey, getTranscription);
+routerV2.post(
+  '/igbo-to-english',
```

Comment:
i think we should name this route to `/translate` to keep it language agnostic for when we want to support translating between other native Nigerian languages